tests/devapps/MauiApps/MauiApps.sln

```diff
@@ -7,7 +7,9 @@ Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "Microsoft.Identity.Client",
 EndProject
 Project("{9A19103F-16F7-4668-BE54-9A1E7A4F7556}") = "MauiAppWithBroker", "MauiAppWithBroker\MauiAppWithBroker.csproj", "{B2CDEFCB-CE8D-4527-8F68-23B1BE6FB3D2}"
 EndProject
-Project("{FAE04EC0-301F-11D3-BF4B-00C04F79EFBC}") = "MauiAppBasic", "MauiAppBasic\MauiAppBasic.csproj", "{D790B117-3C9A-4F69-BF62-8E29EEE3632D}"

```

Comment:
Do we need a new SLN ? Can't you add the MauiApp to the existing LibsAndSamples?